packages/frontend/src/components/Charts/LongChart.tsx

```diff
@@ -296,7 +305,11 @@ export function LongChart() {
               </div>
               <div className={classes.legendContainer}>
                 <div style={{ width: '20px', height: '20px', backgroundColor: '#00E396' }}></div>
-                <div>Squeeth PNL</div>
+                <div>Squeeth Deribit PNL</div>
+              </div>
+              <div className={classes.legendContainer}>
+                <div style={{ width: '20px', height: '20px', backgroundColor: '#FEB01B' }}></div>
```

Comment:
It's better to create a reusable component for 20*20 box which receives background color